template/README.md

```diff
@@ -26,3 +26,92 @@ For jest you will need to add this to your package.json
     }
 }
 ```
+
+## Implementation Notes
+
+With a typical implementation, the consuming application can pull in your shared UI using two lines of code:
+
+```
+<script type="text/javascript" src="https://static-content.mindbodyonline.com/ui/my-shared-ui/0.1.1/app.js"></script>
+```
+Be sure to adjust the name and version to match your shared UI.
+
+```
+<div id="root"></div>
+```
+If you change `root` to another name in `index.tsx`, be sure to change it here as well.
+
+### Passing Parameters
+
+Parameters can be passed into the shared UI a couple of different ways. One option is to set a global variable (`window.siteLocale = 'en';`) in the consuming application and read that variable from the shared UI.
```

Comment:
This new section is great. I think we should make "Getting Data" a section and explaining the three different options. 

First: use an API endpoint
Second: Mounting the app on script load and getting data from a window variable (whether it be a single value, array, or object)
Third: Mounting CRA app from within consuming application